edx_name_affirmation/tests/test_views.py

```diff
@@ -456,6 +457,7 @@ def _get_expected_response(
                 'verified_name': verified_name_obj.verified_name,
                 'profile_name': verified_name_obj.profile_name,
                 'verification_attempt_id': verified_name_obj.verification_attempt_id,
+                'verification_attempt_status': None,
```

Comment:
Could you add a quick test to ensure that the `verification_attempt_status` is correctly exposed via the view? You could just mock out the `verification_attempt_status` property to avoid mocking out the model.